plasmapy/particles/ionization_state.py

```diff
@@ -34,14 +34,14 @@ class IonicLevel:
 
     Parameters
     ----------
-    ion: `~plasmapy.particles.particle_class.ParticleLike`
+    ion : |atom-like|
         The ion for the corresponding ionic fraction.
 
-    ionic_fraction: real number between 0 and 1, optional
+    ionic_fraction : real number, optional
         The fraction of an element or isotope that is at this ionization
-        level.
+        level. Must be between 0 and 1.
```

Comment:
Non-inclusive, I suspect?

Reply:
Good catch — I added a clarification.

Reply:
Oh right, it would be inclusive because you want to be able to represent a pure state. Makes sense 👍🏻